app/lib/features/events/widgets/event_item.dart

```diff
@@ -35,50 +41,35 @@ class EventItem extends StatelessWidget {
           pathParameters: {'calendarId': event.eventId().toString()},
         );
       },
-      child: Card(
-        margin: margin,
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.start,
-          children: [
-            _buildEventDate(context),
-            Expanded(
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.center,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  _buildEventTitle(context),
-                  _buildEventSubtitle(context),
-                ],
-              ),
+      child: Stack(
+        alignment: Alignment.topLeft,
+        children: [
+          Card(
+            margin: margin,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: [
+                EventDateWidget(calendarEvent: event),
+                Expanded(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      _buildEventTitle(context),
+                      Consumer(builder: _buildEventSubtitle),
+                      const SizedBox(height: 4),
+                      if (isShowRsvp) _buildRsvpStatus(context),
```

Comment:
this isn't convincing me yet fully. I feel like we can merge it as is, but a) having a third line and b) a rather textual as well, feels like a bit much. Why not just show the icon itself as the trailing widget?